### ISSUE TITLE
Screen reader does not announce paragraph information on Add card button in browse mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13765]
<img width="487" alt="image" src="https://user-images.githubusercontent.com/93735775/171717106-21868c6e-fac3-40b1-b23d-a345fa3da005.png">

CodePen ->   [A Pen by brandonthomas (codepen.io)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fcodepen.io%2fbrandonthomas%2fpen%2fgOXNONV&data=05%7c01%7cv-knarang%40microsoft.com%7cfdb73c70942f40fbc3b608da1bbffc44%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637852808631632915%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=4FhK9PXgJhPeQ52mPelXY97xCrc6vocaVOZNXLARhR4%3D&reserved=0)

### Steps to reproduce:
1. Open [Admin-Controls](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2fadmincontrols&data=05%7c01%7cv-knarang%40microsoft.com%7cfdb73c70942f40fbc3b608da1bbffc44%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637852808631632915%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=QBBr4bEuDTK8oIwQaUkEPnsuHeD4b5Dk%2BwHbdS6erHg%3D&reserved=0)
2. Open add card panel 
3. Navigate to Dashboard tab and activate it.
4. Navigate to "Add card" button and activate it in Variants section.
5. Navigate to 'Add card' button by tab key and activate through enter key.
 7. Verify that screen reader does announce paragraph information on Add card button in browse mode or not.

### Actual behavior:
Screen reader does not announce paragraph information on Add card button in browse mode. Screen reader announces name of the Add card button.

### Expected behavior:
Screen reader should announce paragraph information also on Add card button in browse mode.

#### NVDA installed/portable/running from source:
Windows 11OS: Version Dev (OS Build 22523.1000)
Edge Dev:  Version 98.0.1108.15(Official build) dev(64-bit)
URL: Admin-Controls

#### NVDA version:
NVDA 2021.3.5

#### Windows version:
Windows 11OS: Version Dev (OS Build 22523.1000)

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Windows 11

OS: Version Dev (OS Build 22523.1000)

Edge Dev:  Version 98.0.1108.15(Official build) dev(64-bit)

URL: [Admin-Controls](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2fadmincontrols&data=05%7c01%7cv-knarang%40microsoft.com%7cfdb73c70942f40fbc3b608da1bbffc44%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637852808631632915%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=QBBr4bEuDTK8oIwQaUkEPnsuHeD4b5Dk%2BwHbdS6erHg%3D&reserved=0)

**Comment by michaelDCurran** [comment]
I don't know what you mean by paragraph information sorry.
Can you please provide clearer steps to reproduce, including the specific speech output you get / what you expect.

**Comment by Adriani90** [comment]
I think here we talk about the aria_describedby property which is applied on the button but cannot be accessed with arrow keys in browse mode.
In my test, when focusing the button in browse mode with arrow keys, NVDA reports "Domains button"

Expected reporting: "Domains button (manage your organisation's domains and monitor their health status)".

**Comment by tgeczy** [comment]
Is there a wider issue referencing the inconsistent behavior when aria-describedby is getting applied and how it may not be read in browse mode? Competing products also behave in similar ways, so this might be something in the Spec itself for name calculations? I'm running into this problem in another code snippet:
"  <div role="group" aria-labelledby="listrow-title-1" aria-describedby="listrow-subtitle-1">
    <button id="listrow-title-1">boy pablo</button>
    <p id="listrow-subtitle-1" hidden>Artist</p>
  </div>
"
This will read the word "artist" after the "grouping," but only when I am in focus mode enabled. Rather than create another issue around that's nearly a duplicate of what's described here, figured it's better to add voice to this one.

**Comment by Adriani90** [comment]
This is a duplicate of #12718. It is valid for all aria describedby cases when using browse mode. NVDA does not report them when tabbing in browse mode, when using arrow keys or quick navigation keys.
Note the comments in the referenced issue for more details.
